e2e/pages/organizations/developers/developers.readonly.spec.js

```diff
@@ -95,3 +102,63 @@ describe('When a user is on the Developer page for a Developer that doesn\'t exi
         expect(browser.getUrl()).toContain('#/search');
     });
 });
+
+describe('when logged in as an Admin', () => {
+    beforeEach(() => {
+        login.logInWithEmail('admin');
+        login.logoutButton.waitForDisplayed();
+    });
+
+    afterEach(() => {
+        login.logOut();
+    });
+
+    describe('when on the "Greenway Health, LLC" Developer page', () => {
+        const name = 'Greenway Intergy Meaningful Use Edition';
+        const productId = 837;
+        let product, version;
+        let timestamp = (new Date()).getTime();
```

Comment:
```suggestion
        let timestamp;
```

Setting the variable to a value should happen in the 'beforeEach' block. That way, if there were multiple `it` blocks covered by the `beforeEach`, they would each have a different value for the timestamp, which helps ensure we're actually using "random-ish" values for these things

Reply:
Did.